src/tagstudio/qt/modals/settings_panel.py

```diff
@@ -135,6 +135,12 @@ def __build_global_settings(self):
             Translations["settings.open_library_on_start"], self.open_last_lib_checkbox
         )
 
+        self.auto_generate_sound_thumbnail = QCheckBox()
```

Comment:
Suggested name change based on the main review feedback
```suggestion
        self.generate_thumbs = QCheckBox()
```